### ISSUE TITLE
Modify direct backscatter calculation in ED RTM

### ISSUE THREAD

**Issue by ashiklom** [EDmodel/ED2#322]
As part of my ED2-PROSPECT work, I was comparing albedo simulations from EDR and SAIL (a common RTM in the remote sensing community) and noticed that, under identical parameters, EDR significantly underestimates albedo compared to SAIL, especially at low solar angles (high `czen` = cosine solar zenith angle).

![edr-sail-comparison-czen](https://user-images.githubusercontent.com/5680221/106921771-5763e400-66da-11eb-99d4-421ee5028650.png)

Based on the literature, it seems this is a known issue with the Sellers model. Specifically, the Sellers approach for calculating single-scattering albedo results in an underestimation of backscatter for direct radiation (for various technical reasons that are nicely described in [Yuan et al. 2017 JAMES](https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1002/2016MS000773), section 3.2, in the text after equation 12), which in turn leads to underestimates of albedo.

Should we consider following the Yuan et al. recommendations to this problem (and a few other modifications to the two-stream scheme) in the canopy RTM? Among other things, this might ameliorate issues with ED2 running too hot (wasn't this mentioned at the meeting? Maybe I'm misremembering...).

**Comment by mpaiao** [comment]
@ashiklom Yes, any development that improves model processes is very welcome!  I would just suggest to implement this as as new option for ICANRAD (as opposed to changing the existing ones).
